app/scripts/services/get-toggles-for-release.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference path="../../../typings/tsd.d.ts" />
+/// <reference path="abstract-http-service.ts" />
+
+module labsFrontendApp
+{
+    export interface Toggle
+    {
+        id: number,
+        name: string
+        summary: string
+    }
+
+    export interface GetTogglesForRelease
+    {
+        execute( releaseId: number ): ng.IPromise<Array<Toggle>>
+    }
+
+    export class HttpGetTogglesForRelease extends HttpService<Array<Toggle>> implements GetTogglesForRelease
+    {
+        protected url = 'toggle/list';
```

Comment:
I guess that should be part of the integration test. 

Do you have the urls available?